forms/Dockerfile

```
@@ -0,0 +1,19 @@
+# 1. Fix the "Could not reliably determine FQDN" warning
+RUN echo "ServerName localhost" >> /etc/apache2/apache2.conf
+
+# 2. Change Apache to listen on 8080 (Matches your Caddyfile)
+RUN sed -i 's/Listen 80/Listen 8080/g' /etc/apache2/ports.conf && \
+    sed -i 's/:80/:8080/g' /etc/apache2/sites-available/000-default.conf
+
+# Install PDO MySQL driver
+RUN docker-php-ext-install pdo pdo_mysql
+
+# (optional but recommended)
+RUN docker-php-ext-enable pdo_mysql
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)